### ISSUE TITLE
Redirection list alias

### ISSUE THREAD

**Issue by khaoula-abaidi** [sympa-community/sympa#1368]
Hello,
I'm using sympa 6.2.40.
I want use my list name test@lists.mydomain.fr  with this alias test@mydomain.fr.
I mean i want use my alias test@mydomain.fr as a sender and when users send to test@mydomain.fr alias the list receive the mails.

I have tried the anonymous_sender value into my list but it not responding for my request.
How can i can do it into list parameters ? or into my postfix configuration ? 
Thank you for your help

**Comment by racke** [comment]
Yes, domain rewriting and manipulating headers is better left to the MTA (Postfix).
